armi/utils/tests/test_units.py

```diff
@@ -73,3 +71,70 @@ def test_getTf(self):
 
         with self.assertRaisesRegex(ValueError, "Tc=0 and Tk=200"):
             units.getTf(Tc=0, Tk=200)
+
+    def test_convertPascalToPascal(self):
+        """This is a pass-through function"""
+        for val in [0.0, -99.141, 123, 3.14159, -2.51212e-12]:
+            self.assertEqual(val, units.convertPascalToPascal(val))
```

Comment:
So this test is fine, but I just peeked at `units.py::convertPascalToPascal`. What is this used for? It doesn't seem particularly useful... 

Doing a grep for `calculatePressure` also returns nothing...? I wonder if instead of adding a test, we can just remove `convertPascalToPascal` altogether with... 🤔

Reply:
Yeah, I definitely considered that. But the docstring convinced me there was a good use for this method.

I just did some digging, and I couldn't find a single downstream repo that uses this dubious method.

You're right; I like the idea of deleting it better.

Reply:
Sadly, I _did_ find downstream use-cases of `PRESSURE_CONVERTERS`, which use `convertPascalToPascal`.

BUT, sure, I replaced `convertPascalToPascal` with an inline lambda, since all anyone needs is `PRESSURE_CONVERTERS` and not `convertPascalToPascal`.

Good call!

Reply:
Wow!!! Nice work